tools/CoherenceBuild/CoherenceBuild.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <RuntimeFrameworkVersion>1.0.5</RuntimeFrameworkVersion>
```

Comment:
Is this required by the tool? If so, might be easier to bump it up to 2.0.0. Think that might be pretty straightforward.